include/atari.h

```diff
@@ -261,11 +261,11 @@ extern void atrx15p2_tgi[];
 #define AT_PAL      1
 
 /* valid _dos_type values */
-#define ATARIDOS    0
-#define SPARTADOS   1
-#define OSADOS      2
-#define MYDOS       3
-#define XDOS        4
+#define SPARTADOS   0
+#define OSADOS      1
+#define XDOS        2
+#define ATARIDOS    3
+#define MYDOS       4
```

Comment:
A valid question IMHO. I *think* the original arrangement reflected more or less the historical development (although I am not sure), which makes it "better" for me than an arbitrary arrangement. Besides, are we sure that this change doesn't break existing code? To my Atari-untrained eye it looks like it might, therefore justification for this change is even more important.